terminal/location/client.go

```diff
@@ -52,6 +52,19 @@ func (c Client) Update(id string, params *stripe.TerminalLocationParams) (*strip
 	return location, err
 }
 
+// Del removes a location.
+func Del(id string, params *stripe.TerminalLocationParams) (*stripe.TerminalLocation, error) {
```

Comment:
I assume we are okay with sharing Params here having fields more than what apirefs now has..

Reply:
For now we're sharing in go for some of those. So I'd rather stay consistent